janino/src/main/java/org/codehaus/janino/UnitCompiler.java

```diff
@@ -11748,21 +11748,21 @@ private int getTargetVersion() {
      * @param opIdx One of {@link #EQ}, {@link #NE}, {@link #LT}, {@link #GE}, {@link #GT} or {@link #LE}
      */
     private void
-    ifNumeric(Locatable locatable, int opIdx, Offset dst) {
+    ifNumeric(Locatable locatable, int opIdx, Offset dst, boolean orientation) {

```

Comment:
please include a javadoc comment, explaining what the parameter is for

Reply:
sure, updated.